uat/testing-features/src/main/resources/greengrass/components/recipes/client_java_sdk.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: aws.greengrass.client.Mqtt5JavaSdkClient
+ComponentVersion: '1.0.0'
+ComponentDescription: MQTT5 Java Client powered by AWS Java SDK
+ComponentPublisher: Amazon
+ComponentConfiguration:
+  DefaultConfiguration:
+    agentId: agent1
+    broker:
+      address: 127.0.0.1
+      port: 47619
+Manifests:
+  - Artifacts:
+      - URI: classpath:/greengrass/artifacts/aws.greengrass.client.Mqtt5JavaSdkClient.jar
+        Permission:
+          Read: ALL
+          Execute: ALL
+    Lifecycle:
+      Startup: |
```

Comment:
just changing this to startup without changing the code is not correct. Leave it as run until the code is updated.

Reply:
Fixed d2ca07c1c3a20331a116f61e1e820d4553066829